data/net/minecraft/network/chat/MessageSignature.mapping

```diff
@@ -3,9 +3,13 @@ CLASS net/minecraft/network/chat/MessageSignature
 		ARG 1 buffer
 	METHOD equals (Ljava/lang/Object;)Z
 		ARG 1 other
+	METHOD verify (Lnet/minecraft/util/SignatureValidator;Lnet/minecraft/network/chat/SignedMessageHeader;Lnet/minecraft/network/chat/SignedMessageBody;)Z
+		ARG 1 validator
+		ARG 2 header
+		ARG 3 body
 	METHOD verify (Lnet/minecraft/util/SignatureValidator;Lnet/minecraft/network/chat/SignedMessageHeader;[B)Z
-		ARG 1 signatureValidator
+		ARG 1 validator
 		ARG 2 header
-		ARG 3 bodyDigest
+		ARG 3 body
```

Comment:
Idk if this is right

Reply:
``body`` should be hashBytes.
For reference see: ``SignedMessageValidator#validateMessage(PlayerChatMessage)``